cou/apps/base.py

```diff
@@ -739,11 +739,15 @@ def _get_wait_step(self) -> PostUpgradeStep:
         """
         if self.wait_for_model:
             description = (
-                f"Wait {self.wait_timeout}s for model {self.model.name} to reach the idle state."
+                f"Wait for up to {self.wait_timeout}s for model {self.model.name} "
+                "to reach the idle state"
             )
             apps = None
         else:
-            description = f"Wait {self.wait_timeout}s for app {self.name} to reach the idle state."
+            description = (
+                f"Wait for up to {self.wait_timeout}s for app {self.name} "
```

Comment:
Can you use `'` for app name.

Reply:
Good point. I updated a new patch addressing all quotation mark alignment.